### ISSUE TITLE
Lots of copies of big-buck-bunny?

### ISSUE THREAD

**Issue by larsbergstrom** [servo/servo#12695]
@gsnedders Do we need so many copies of the big-buck-bunny movie file? Or is there some way to reduce the duplication a bit?

After some complaints from the Gecko side (passed on via @bholley) about a "lot of repeated binary files in the CSS tests takin' up a lot of disk space", I did a `find . -type f -size +1M | xargs du -k | sort -n` and definitely noticed that there are a few duplicated files that are > 1MB.

```
1756    ./css-regions-1_dev/html/reference/support/big-buck-bunny-240p.mp4
1756    ./css-regions-1_dev/html/support/big-buck-bunny-240p.mp4
1756    ./css-regions-1_dev/xhtml1/reference/support/big-buck-bunny-240p.mp4
1756    ./css-regions-1_dev/xhtml1/support/big-buck-bunny-240p.mp4
1756    ./css-regions-1_dev/xhtml1print/reference/support/big-buck-bunny-240p.mp4
1756    ./css-regions-1_dev/xhtml1print/support/big-buck-bunny-240p.mp4
1756    ./css-transforms-1_dev/html/reference/support/big-buck-bunny-240p.mp4
1756    ./css-transforms-1_dev/html/support/big-buck-bunny-240p.mp4
1756    ./css-transforms-1_dev/xhtml1/reference/support/big-buck-bunny-240p.mp4
1756    ./css-transforms-1_dev/xhtml1/support/big-buck-bunny-240p.mp4
1756    ./css-transforms-1_dev/xhtml1print/reference/support/big-buck-bunny-240p.mp4
1756    ./css-transforms-1_dev/xhtml1print/support/big-buck-bunny-240p.mp4
2104    ./css21_dev/html4/chapter-8.htm
2104    ./css21_dev/xhtml1/chapter-8.xht
2104    ./css21_dev/xhtml1print/chapter-8.xht
2416    ./css-regions-1_dev/html/reference/support/big-buck-bunny-240p.ogv
2416    ./css-regions-1_dev/html/support/big-buck-bunny-240p.ogv
2416    ./css-regions-1_dev/xhtml1/reference/support/big-buck-bunny-240p.ogv
2416    ./css-regions-1_dev/xhtml1/support/big-buck-bunny-240p.ogv
2416    ./css-regions-1_dev/xhtml1print/reference/support/big-buck-bunny-240p.ogv
2416    ./css-regions-1_dev/xhtml1print/support/big-buck-bunny-240p.ogv
2416    ./css-transforms-1_dev/html/reference/support/big-buck-bunny-240p.ogv
2416    ./css-transforms-1_dev/html/support/big-buck-bunny-240p.ogv
2416    ./css-transforms-1_dev/xhtml1/reference/support/big-buck-bunny-240p.ogv
2416    ./css-transforms-1_dev/xhtml1/support/big-buck-bunny-240p.ogv
2416    ./css-transforms-1_dev/xhtml1print/reference/support/big-buck-bunny-240p.ogv
2416    ./css-transforms-1_dev/xhtml1print/support/big-buck-bunny-240p.ogv
2592    ./css21_dev/html4/reftest-toc.htm
2616    ./css21_dev/xhtml1/reftest-toc.xht
2616    ./css21_dev/xhtml1print/reftest-toc.xht
3216    ./css-regions-1_dev/html/reference/support/big-buck-bunny-240p.webm
3216    ./css-regions-1_dev/html/support/big-buck-bunny-240p.webm
3216    ./css-regions-1_dev/xhtml1/reference/support/big-buck-bunny-240p.webm
3216    ./css-regions-1_dev/xhtml1/support/big-buck-bunny-240p.webm
3216    ./css-regions-1_dev/xhtml1print/reference/support/big-buck-bunny-240p.webm
3216    ./css-regions-1_dev/xhtml1print/support/big-buck-bunny-240p.webm
3216    ./css-transforms-1_dev/html/reference/support/big-buck-bunny-240p.webm
3216    ./css-transforms-1_dev/html/support/big-buck-bunny-240p.webm
3216    ./css-transforms-1_dev/xhtml1/reference/support/big-buck-bunny-240p.webm
3216    ./css-transforms-1_dev/xhtml1/support/big-buck-bunny-240p.webm
3216    ./css-transforms-1_dev/xhtml1print/reference/support/big-buck-bunny-240p.webm
3216    ./css-transforms-1_dev/xhtml1print/support/big-buck-bunny-240p.webm
```

**Comment by notriddle** [comment]
I guess we can't use hardlinks?

**Comment by gsnedders** [comment]
I'll check, but I'm not sure how easily we can get rid of this till the build system is gone. :\

There again, at least on the Servo side once we move to running tests directly from csswg-test we should avoid a fair bit of the duplication (simply because there aren't the multiple built formats!).

That said, they shouldn't be bloating the git repo by too much because they'll share object files. It's only checkouts which they really bloat.

**Comment by nox** [comment]
We still have many bunnies.

**Comment by retep998** [comment]
🐇🐇🐇

**Comment by gsnedders** [comment]
This should be resolved when #16175 is fixed (they're gone upstream).

**Comment by jdm** [comment]
All gone!
